tests/sentry/grouping/ingest/test_seer.py

```diff
@@ -147,7 +153,14 @@ def test_obeys_customized_fingerprint_check(self) -> None:
             data={
                 **self.event_data,
                 "fingerprint": ["failedtofetcherror"],
-                "_fingerprint_info": {"matched_rule": {"is_builtin": True}},
+                "_fingerprint_info": {
+                    "matched_rule": {
+                        "is_builtin": True,
+                        "matchers": [["family", "python"]],
+                        "fingerprint": ["failedtofetcherror"],
+                        "text": "text",
+                    }
```

Comment:
Nit: This is slightly more realistic data.

```suggestion
                    "matched_rule": {
                        "is_builtin": True,
                        "matchers": [["type", "FailedToFetchError"]],
                        "fingerprint": ["failedtofetcherror"],
                        "text": 'type:"FailedToFetchError" -> "failedtofetcherror"',
                    }
```

Reply:
good to know! thank you!